frontend/src/CrosswordSelector.css

```diff
@@ -0,0 +1,48 @@
+.crosswordselector {
+  width: 100%;
+  max-width: 1024px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 348px);
```

Comment:
This causes some weird alignment when having 3 or more crosswords. Why is this preferd over `repeat(auto-fit, minmax(300px, 1fr)`? This seemed to fix the alignment with multiple crosswords with the help of `justify-items: center` and removing the margin on the cards.

Reply:
Good observation, fixed 👍